src/legacy/components/site-header/_site-header.scss

```diff
@@ -82,9 +82,9 @@
 
     .site-branding--icon {
       background-image: svg-load('icons/mltshp-icon.svg');
-      display: inline-block;
-      height: 2.475em; // approximated this comparing to label mixin height
-      width: 2.475em;
+      display: block;
```

Comment:
I guess we could just change the .site-branding--icon element to a div and drop this property. That will require a small change to MLTSHP (`templates/base.html`).

Reply:
Good call. Made the change.